plaso/parsers/__init__.py

```diff
@@ -35,6 +35,7 @@
 from plaso.parsers import mac_wifi
 from plaso.parsers import mactime
 from plaso.parsers import mcafeeav
+from plaso.parsers import mobile_installation
```

Comment:
I would suggest to use a more specific module name here, such as ios_mobile_installation_logs or ios_mobile_installation